Demo/UITests/BraintreeDropIn_UITests.swift

```diff
@@ -700,151 +705,6 @@ class BraintreeDropIn_PayPal_Disabled_UITests: XCTestCase {
     }
 }
 
-class BraintreeDropIn_ThreeDSecure_UITests: XCTestCase {
```

Comment:
These are 3DSv1 tests - the v2 tests still exist below but we do not need these tests because the legacy flow is no longer supported by Cardinal